main.js

```diff
@@ -55,11 +55,28 @@
         });
     }
 
+    //Displays and hides the sidebar for mobile
+    function displaySidebar() {
+      document.getElementById("sidebar").style = "display:block";
+      document.getElementById("mobileMenuIcon").style = "display:none";
+      document.getElementById("mobileHideMenu").style = "display:block";
+    }
+
+    function hideSidebar() {
+      document.getElementById("sidebar").style = "";
```

Comment:
since we're converting this to being one element that simply changes the direction of the arrow, we need to:
1. Display the sidebar
2. Change the element to having the opposite arrow (via `id` substitution, ideally, rather than setting `.style` directly)
3. Change the element's `onclick` to `displaySidebar`